crates/router/src/connector/bankofamerica/transformers.rs

```diff
@@ -1321,11 +1322,11 @@ pub enum BankOfAmericaAuthEnrollmentStatus {
 #[serde(rename_all = "camelCase")]
 pub struct BankOfAmericaConsumerAuthValidateResponse {
     ucaf_collection_indicator: Option<String>,
-    cavv: Option<String>,
-    ucaf_authentication_data: Option<String>,
+    cavv: Option<Secret<String>>,
```

Comment:
@AkshayaFoiger CAVV and XID are authentication results for 3ds transaction, hence these need not to be masked